server/v2/stf/stf.go

```diff
@@ -22,6 +22,8 @@ import (
 // Identity defines STF's bytes identity and it's used by STF to store things in its own state.
 var Identity = []byte("stf")
 
+var theExecutionContext = struct{}{}
```

Comment:
Linter will complain that we shouldn't use empty struct for avoiding collision. We should create a type alias to an empty struct and use that as key.